test/uk/gov/hmrc/emcstfe/fixtures/GetMovementFixture.scala

```diff
@@ -369,26 +370,31 @@ trait GetMovementFixture extends BaseFixtures with TraderModelFixtures {
-                                               |              <body:NotificationDateAndTime>2001-12-17T09:30:47.0Z</body:NotificationDateAndTime>
-                                               |              <body:AdministrativeReferenceCode>13AB1234567891ABCDEF9</body:AdministrativeReferenceCode>
-                                               |              <body:SequenceNumber>1</body:SequenceNumber>
-                                               |            </body:ExciseNotification>
-                                               |          </body:NotificationOfDivertedEADESAD>
-                                               |        </body:Body>
-                                               |      </body:IE803>
+                                               |      <urn:IE803 xmlns:ie803="urn:publicid:-:EC:DGTAXUD:EMCS:PHASE4:IE803:V3.13">
+                                               |        <urn:Header>
+                                               |          <urn:MessageSender>NDEA.GB</urn:MessageSender>
+                                               |          <urn:MessageRecipient>NDEA.GB</urn:MessageRecipient>
+                                               |          <urn:DateOfPreparation>2020-12-03</urn:DateOfPreparation>
+                                               |          <urn:TimeOfPreparation>13:36:43.326</urn:TimeOfPreparation>
+                                               |          <urn:MessageIdentifier>GB100000000289576</urn:MessageIdentifier>
+                                               |        </urn:Header>
+                                               |        <urn:Body>
+                                               |          <urn:NotificationOfDivertedEADESAD>
+                                               |            <urn:ExciseNotification>
+                                               |              <urn:NotificationType>1</urn:NotificationType>
+                                               |              <urn:NotificationDateAndTime>2024-06-05T00:00:01</urn:NotificationDateAndTime>
+                                               |              <urn:AdministrativeReferenceCode>20GB00000000000341760</urn:AdministrativeReferenceCode>
+                                               |              <urn:SequenceNumber>1</urn:SequenceNumber>
+                                               |            </urn:ExciseNotification>
+                                               |            <urn:DownstreamArc>
+                                               |              <urn:AdministrativeReferenceCode>$testArc</urn:AdministrativeReferenceCode>
+                                               |            </urn:DownstreamArc>
+                                               |            <urn:DownstreamArc>
+                                               |              <urn:AdministrativeReferenceCode>${testArc}1</urn:AdministrativeReferenceCode>
```

Comment:
Appending a 1 to that `testArc` would push it to 22 characters which is 1 over the schema defined limit although this is probably okay for display purposes.

```  <!--==========================================================-->
  <!--===                    Upstream ARC                    ===-->
  <!--==========================================================-->
  <xs:simpleType name="UpstreamArcType">
    <xs:annotation>
      <xs:documentation>
        <doc:description value="Upstream ARC" />
      </xs:documentation>
    </xs:annotation>
    <xs:restriction base="xs:token">
      <xs:length value="21" />
      <xs:pattern value=".{21}" />
    </xs:restriction>
  </xs:simpleType>```

Reply:
Also I wouldn't have thought a notification type 1 would have downstream arcs populated, isn't that more likely for a notification type 2.

Reply:
Although we can only have 1 `ie803:IE803` so I suppose it caters for both types of IE803's
<img width="1048" alt="image" src="https://github.com/hmrc/emcs-tfe/assets/11269301/e00a68c4-65dd-4970-bcb2-aab7d34003bc">